tests/samples/songkick/elysianfields.html

```diff
@@ -585,7 +585,7 @@ <h2>Posters (8)</h2>
 </div>
 
           <div class="component events-summary" id="gigography-summary">
-  <h2 class="calendar">Past concerts (321) <small><a rel="nofollow" href="/artists/236156-elysian-fields/gigography">See all</a></small></h2>
+  <h2 class="calendar">Past concerts (321) <small><a rel="nofollow" href="/artists/236156-elysian-fields/gigography">See all</a></small></h2>
```

Comment:
This is reporting like a strange change in a space. Maybe a encoding issue. Could you just revert this file to the master state so that this space is as it was?.

Reply:
I am not very comfortable with git and not sure how to do this, can you help me with this? I am working with the GitHub GUI.

I originally made my changes to `elysianfields.html`, then renamed it to `elysianfields_1.html`, and created a new file `elysianfields.html` by copying the code from the master branch.

Reply:
I tried to do in this commit https://github.com/scrapinghub/extruct/pull/139/commits/66c7b89e57ae2f3c134b5dd90e20ad6f7def45f9 but Github still reports a difference. Anyway, this is not important, so let's forget about it :+1: